src/include/index/bwtree.h

```diff
@@ -346,127 +303,123 @@ class BwTreeBase {
    * This function must be called when the garbage pool is empty
    */
   void DestroyThreadLocal() {
-    bwt_printf("Destroy %lu thread local slots\n", thread_num);
-    
+    LOG_TRACE("Destroy %lu thread local slots\n", thread_num);
```

Comment:
Remove the newline character after all of these new LOG_TRACE

Reply:
Sorry I will fix it soon.